src/main/java/websocket/WSManager.java

```diff
@@ -236,6 +242,14 @@ public void handleMessageSendError(String message) {
     public void setAuthInfo(String userID, String userToken) {
         this.userID = userID;
         this.userToken = userToken;
+        this.sendAllAuthenticatedRequests();
+    }
+
+    // TODO: fix potential concurrency issue with the list?
```

Comment:
Ok. I knew this would need to be changed before it was accepted, but wasn't quite sure how. Should it be synchronized on WSManager.class?